lib/assets/package_application_patches/codesign_args_patch.diff

```diff
@@ -0,0 +1,33 @@
+From 13f2c76fd11b0d861ddf205d4d6ccc0c45a0d4b2 Mon Sep 17 00:00:00 2001
```

Comment:
Given that the patches will be reapplied based on the order given by Dir, it is usually a good practice to keep files automatically sorted given their natural order. Hence git format-patch creating files with a 000x prefix. This avoids trying to reapply patches in a different order, which could cause failures.